backend/main.py

```diff
@@ -58,6 +58,60 @@ def main():
     dps = round(player.calc_damage(potency, my_comp), 2)
     return jsonify({"dps": dps})
 
+@app.route('/update_stats', methods=["POST"])
+def update_stats():
+    """Calculates damage based on input. Accepts and returns JSON. JSON format is as follows:
```

Comment:
doc string should be updated to say that json returns gcd time and mp consumption